Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -1,6 +1,8 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "extend/os/pkgconf"
```

Comment:
```suggestion
require "extend/os/mac/pkgconf"
```

that's a better file path for this